utils/rth

```diff
@@ -165,7 +165,8 @@ class ResilienceTest(object):
             config2_lower = config2.lower()
             output_obj = os.path.join(self.tmp_dir,
                                       config1_lower + '_' + config2_lower)
-            tmp_dir_contents = glob.glob(os.path.join(self.tmp_dir, '*', '*'))
+            tmp_dir_contents = glob.glob(
+                os.path.join(self.tmp_dir, self.config_dir_map[config], '*'))
```

Comment:
The only reason for this is to force the `target_run` command to copy the libraries over, so I guess this works. You do need to use the correct configuration here, though (`config` doesn't exist). You can see what's being tested in the `link` function: what happens when you statically link against one library, but dynamically use a potentially different library at run time?

Reply:
Thanks for catching that. I must have typed that wrong in the computer I used to send the diff. I will fix it immediately.